README.rst

```diff
@@ -807,6 +807,12 @@ Choose from the list of available rules:
   Configuration options:
 
   - ``exclude`` (``array``): list of functions to ignore; defaults to ``[]``
+  - ``include``: list of function names or sets to fix. Defined sets are
+    ``@internal`` (all native functions), ``@all`` (all global functions) and
+    ``@compiler_optimized`` (functions that are specially optimized by Zend);
+    defaults to ``['@internal']``
+  - ``scope`` (``'global'``, ``'namespaced'``): fix function calls only if done in "user
```

Comment:
I'd replace `in  "user defined namespaces"` (as it may suggest there might be `not user defined namespaces` or something) with simply `inside a namespace`.

Also when I put "function calls done inside namespace" into google translate, translated it to some language, then back to English it came back with "made" instead of "done".

Reply:
:+1: updated